### DISCUSSION TITLE
Problem with messages ending in spam

### DISCUSSION THREAD

**Discussion by miroslavskela** [postalserver/postal#1674] [Help with using Postal]
Hello, tnx again for this great product.
I have some doubts when sending email using postal. My postal server sends some transactional mails for about 2 weeks, and everything looks good on postal side, but clients say that they didn't received mails or mails are in spam. Today I did some testing using glockapps.com and half of mails that I send from my postal server, with registered domain ends in spam, (all outlook and gmail messages). When I registered same domain on other email delivery service almost all messages ends in inbox. Does anyone know why this happens, and If there is some chance to improve current situation with my postal server. When checking my mail on mail-tester score is 10/10. That another email delivery service, even when i register domain that has A record points to ip address that is on many blacklists, by glockapps report it succesfully delivers emails to inbox. If you need some additional info, let me know. 
Kind Regards.

**Comment by uhlhosting** [comment]
Most important is to check your Sender IP in as DNSBL. And if its clean, do a mail server warmup.

**Reply by miroslavskela** [reply]
Thank you @uhlhosting. I checked it, and it is clean. We have done some kind of warm up, even at this moment we dont send more then 100 transactional mails per day. Also some clients have moved mails from spam to inbox, but problem with gmail and outlook still exist in 90 percent of mails. I have one more server that is even younger then this one, and i use another domain there, and from that server all mails to gmail ends in inbox, with outlook we face same issue there. I have done testing with zerobounce, and noticed some strange warnings and errors there. If you, or someone i interested to check it, I can share it here. Thank you very much, again.

**Reply by miroslavskela** [reply]
Also noticed that when i send mail from usual address for example usualaddress@domain.com mail is in spam, but when mail is send from someothername@domain.com it is in inbox.

**Comment by willpower232** [comment]
Unfortunately outlook/microsoft in particular is quite randomly strict on whether your email is considered spam and whether the recipient will even get it in their junk email or it will just disappear forever.

If all the different kinds of transactional email messages are 10/10 on mail-tester.com then the best you can do is ask your recipients to add your email address to their allow list, Postal did everything it could already and unfortunately email is never a perfect experience.

**Reply by miroslavskela** [reply]
Tnx @willpower232, at the end, I figured it out, that in some situations There is no much I can do. I just one to ask one more question. Did some testing and noticed this error, if you or someone knows how this should be fixed. I set reverse dns to point to mydomain.com and also set hostname mydomain.com on my server but still have this problem.
![Screenshot from 2021-10-31 16-35-14](https://user-images.githubusercontent.com/36046629/139591321-d980751b-bd2e-4159-88c8-cc14dc2b2eb2.png)
.

**Reply by willpower232** [reply]
You will need to ask your server provider how you can configure the reverse DNS record for the IP address they have given you. Some providers (DigitalOcean) do this automatically, other providers (AWS) require you to contact them to set it up.

This may be responsible for some of the rejection of some of your messages.

**Reply by miroslavskela** [reply]
Yes, that's the thing that confuses me, because i set reverse dns on my server provider, it was not something complicated. But it looks like that's not enough. Or something is not correct. Tried to find some info, but it  is confusing also. Some responses say that reverse dns should point to subdomain.domain.com. Other ones say that should point to domain.com as I did.
![Screenshot from 2021-10-31 17-00-16](https://user-images.githubusercontent.com/36046629/139592324-c00680ec-74b9-4372-8a79-f8073a5f6612.png)

**Reply by uhlhosting** [reply]
The PTR records are broken or takes time to propagate from the provider.

**Reply by willpower232** [reply]
You'll have to ask your server provider in case something is wrong.

**Comment by foxmadr** [comment]
I have the very same problem. Please, have you come up with a solution?
I've tried everything without success.
The simple step to reproduce it is to send an email to some gmail account by "Send Message" in Postal web.

**Reply by willpower232** [reply]
Have you tried mail-tester.com and mxtoolbox.com to see if there are any issues?

**Reply by foxmadr** [reply]
Thank you for hint.
mail-tester.com is ok
mxtoolbox.com 0 errors, 11 warnings all seem to be connected with incoming emails but I am going to analyze all of them
SPF, DKIM, DMARC all passed on gmail

I'm wondering if the issue may be connected with our configuration. We use Postal just for sending emails via API for our domain, SMTP is blocked on the mail server. All incomming emails to our domain are handled by googlemail. It's strange because it used to work correctly a few months ago.

**Reply by willpower232** [reply]
it is possible that if the SMTP is blocked then that might be what is upsetting gmail, it cannot validate that Postal is there by SMTP so is suspicious

**Comment by foxmadr** [comment]
It seems to be solved. When I disabled IPv6 on my mai; server, emails are not going to Spam. I'd say that there is some issue in gmail. I have contacted google support for help as all email checks are ok and the only difference is IPv4 vs IPv6.
